cypress/integration/settings/circulation/fee-fine/overdue-fine-policies.spec.js

```diff
@@ -0,0 +1,29 @@
+import devTeams from '../../../../support/dictionary/devTeams';
+import testTypes from '../../../../support/dictionary/testTypes';
+import OverdueFinePolicies from '../../../../support/fragments/settings/circulation/fee-fine/overdueFinePolicies';
+import SettingsMenu from '../../../../support/fragments/settingsMenu';
+
+describe('ui-circulation-settings: overdue fine policies management', () => {
+  beforeEach('Log in', () => {
+    cy.loginAsAdmin({ path: SettingsMenu.circulationoVerdueFinePoliciesPath, waiter: OverdueFinePolicies.waitLoading });
+  });
+
+  it('C5557: Verify that you can create/edit/delete overdue fine policies', { tags: [devTeams.spitfire, testTypes.smoke] }, () => {
+    // TODO add check that name is unique
+    OverdueFinePolicies.openCreatingForm();
+    OverdueFinePolicies.checkCreatingForm();
+    OverdueFinePolicies.checkOverDueFineInCreating();
+    OverdueFinePolicies.fillGeneralInformation('1.00', '2.00', '3.00', '4.00');
```

Comment:
Please moved these values to the variables or object properties. You are using the same values more than one time.

Reply:
I still can find this situation. Please move these values to variables.